runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsConfig.java

```diff
@@ -369,4 +370,26 @@ default boolean groundItemTimers()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "profitValueColor",
+		name = "Profitable high alch items color",
+		description = "Configures the color for profitable high alch items",
+		position = 29
+	)
+	default Color profitValueColor()
+	{
+		return Color.decode("#ffff00");
+	}
+
+	@ConfigItem(
+		keyName = "highlightProfitableOverValue",
+		name = "High alch Profit > Value",
+		description = "Highlights ground items with a high alch profit greater than value. (0 = Disabled)",
+		position = 30
+	)
+	default int getHighlightProfitOverValue()
```

Comment:
I'm not sure how many users would prefer this to be an int config instead of a boolean. If it stays as-is, we should add a "gp" unit (see #10662) and use that here.

Reply:
Left it as a gp value for now. If more feel it should just be a boolean I can change.